include/Game/NPC/TalkMessageCtrl.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include "Game/LiveActor/LiveActor.hpp"
 #include "Game/NPC/TalkMessageFunc.hpp"
+#include "Game/NPC/TalkMessageInfo.hpp"
```

Comment:
Prefer forward declarations for types used with pointer member variables or function parameters.